inc/TRestGeant4VetoAnalysisProcess.h

```diff
@@ -23,10 +23,10 @@
 #ifndef RestCore_TRestGeant4VetoAnalysisProcess
 #define RestCore_TRestGeant4VetoAnalysisProcess
 
-#include <TRestGeant4Event.h>
-#include <TRestGeant4Metadata.h>
+#include <TRestEventProcess.h>
 
-#include "TRestEventProcess.h"
```

Comment:
I am missing something? TRestEventProcess.h doesn't belong to our repo shouldn't be #include "TRestEventProcess.h"

Reply:
I am not clear on the guidelines, but it doesn't belong to geant4lib, it belongs to framework repo, which is a different repo. What should we use in this case?

Reply:
I think we should use angle brackets for standard (system) includes and quotes for libraries belonging to our software.

https://stackoverflow.com/questions/3162030/difference-between-angle-bracket-and-double-quotes-while-including-heade

Reply:
Okay, the most important thing is that we all do the same.

I also read that post and that is why I was doing it this way, since it will be looking into the include directories (`${REST_INCLUDE_DIRS}`) for the REST libraries, that is why I think it should be with <>. What do you think @jgalan ?

Reply:
I think in our case doesn't matter. I remember struggling with this the first c projects that I wrote, basically because one would not do a proper installation at a final installation directory. Then one was needing to define the headers with the quotes so that the compilation system would look for the local directory where the compilation command was executed.

Therefore, the quotes could still be used as a nomenclature to say that it belongs to us, but we usually are certain it belongs to our project because it starts by `TRestXX`.

I could follow the quotes approach in the future, anyhow.

Reply:
In other words, in our case, REST libraries are becoming system libraries.